src/Pages/CreateQuestion.tsx

```diff
@@ -1,35 +1,97 @@
 import styled from '@emotion/styled'
 import { FillBtn } from '../Components/basic/button/Button'
-import { TextInput } from '../Components/basic/InputBox'
-import { Label } from '../Components/basic/Label'
-import { CategoryInput } from '../Components/CategoryInput'
 import { TextEditor } from '../Components/TextEditor'
+import { useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { EditorState, convertToRaw } from 'draft-js';
+import { useSelector } from 'react-redux';
+import { RootState } from '../state/store';
+import axios from 'axios';
 import { palette } from '../styles/theme'
+import ObjectID from 'bson-objectid';
 
 export function CreateQuestion() {
-  function getCategory(cats: string[]) {
-    console.log(cats)
+  const navigate = useNavigate();
+  const cid = useParams().cid;
+  const uid = useSelector((state:RootState)=>state.userInfo._id)
+  const [answer, setAnswer] = useState("");
+  const [objective,setObjective] = useState("")
+  const [explanation, setExplanation] = useState<EditorState>(() => EditorState.createEmpty());
+  const [question, setQuestion] = useState<EditorState>(() => EditorState.createEmpty());
+
+  function updateObjective(e:React.ChangeEvent<HTMLInputElement>){
+    setObjective(e.target.value)
+  }
+
+  function updateAnswer(e:React.ChangeEvent<HTMLInputElement>){
+    setAnswer(e.target.value)
   }
 
+  function submitStem(){
+		const qstemObj = {
+			uid: ObjectID(uid),
+			stem_text: JSON.stringify(convertToRaw(question.getCurrentContent())),
+			raw_string: question.getCurrentContent().getPlainText("\u0001"),
+			explanation: JSON.stringify(convertToRaw(explanation.getCurrentContent())),
+			action_verb: [],
+			keyword: [],
+			cid: cid && ObjectID(cid),
+			options: [],
+			optionSets: [],
+			learning_objective: objective,
+		};
+
+		const rawString = qstemObj.raw_string;
+		const wordcount = rawString.split(" ").filter((word) => word !== "").length;
+		if (rawString === null || wordcount < 1) {
+			alert("Please enter a question.");
+			return;
+		}
+		if (answer === null || answer.match(/^\s*$/) !== null) {
+			alert("Please enter an answer.");
+			return;
+		}
+		if (qstemObj.learning_objective === null) {
+			alert("Please enter learning objective.");
+			return;
+		}
+		axios
```

Comment:
again, use Post function from `utils` since plain axios does not check type for us.